README.md

```diff
@@ -128,7 +128,7 @@ Triage is developed at the University of Chicago's [Center For Data Science and
 
 ## Major Components Used by Triage
 
-Triage makes use of many core data science components developed at DSaPP. These components can be useful in their own right, and are worth checking out if 
+Triage makes use of many core data science components developed at DSaPP. These components can be useful in their own right, and are worth checking out if you'd like to be cool like that:
```

Comment:
I think a good version of this sentence would be:
"if you'd like to try out a subset of Triage's functionality in an existing pipeline".